part1/anecdote/src/App.jsx

```diff
@@ -0,0 +1,59 @@
+import { useState } from 'react'
+
+const Button = ({onClick, text}) => <button onClick={onClick}>{text}</button>
+
+const Display = ({anecdotes, points}) => {
+  return (
+    <div>
+      <p>{anecdotes}</p>
+      <p>has {points} points</p>
+    </div>
+  )
+}
+
+const App = () => {
+
+  function getRandomInt(max) {
+    return Math.floor(Math.random() * max);
+  }
+
+  const handleNextAnecdote = () => {
+    return setSelected(getRandomInt(8))
+  }
+
+  const handleVote = () => {
+    const copy = [...points]
```

Comment:
this is weird, either put the points useState above this function or or pass it in as a parameter, the same for selected below